custom_components/plex_assistant/localize.py

```diff
@@ -121,6 +121,110 @@
             ],
         },
     },
+    "sv": {
+        "play": "spela",
+        "on_the": "på",
+        "not_found": "not found",
+        "cast_device": "cast enhet",
+        "movies": [
+            "film",
+        ],
+        "shows": [
+            "avsnitt",
+            "tv",
+            "serie",
+        ],
+
+        "play_start": [
+            "spela filmen",
+            "spela film",
+            "spela tv programmet",
+            "spela serien",
+            "spela serie",
+            "spela tv",
+            "spela",
+        ],
+
+        "ordinals": {
+            "första": "1",
+            "andra": "2",
+            "tredje": "3",
+            "fjärde": "4",
+            "femte": "5",
+            "sjätte": "6",
+            "sjunde": "7",
+            "åttonde": "8",
+            "nionde": "9",
+            "tionde": "10",
+            "pre": [],
+            "post": [],
+        },
+
+        "season": {
+            "keywords": [
+                "season",
+            ],
+            "pre": [
+                "a",
+                "e",
+            ],
+            "post": [
+                "nummer",
+                "av",
+            ],
+        },
+        "episode": {
+            "keywords": [
+                "episod",
+            ],
+            "pre": [
+                "a",
+                "e",
+            ],
+            "post": [
+                "nummer",
+                "av",
+            ],
+        },
+        "latest": {
+            "keywords": [
+                "senaste",
+                "nyligen",
+                "ny",
+            ],
+            "pre": [
+                "the",
```

Comment:
I think you may have missed this one. Using Google translate on the phrase "play the latest episode of" I get "spela det senaste avsnittet av". So, the post words seem good here, should the pre word for this be "det" and should that also be applied to unwatched, ondeck, season, and episodes also?

Reply:
```suggestion
                "det",
```

You are totally right, it should be "det".

Reply:
Would it make sense to add this "pre" to other things like unwatched, ondeck, season, and episodes or is that not something that would usually be said?

Reply:
Hmm, I don't think that will work,

It would be two different "pre". **Den/Det**.

Reply:
So should the pre for all be:
```
            "pre": [
                "den",
                "det",
             ]
```
or should just den or det be applied to certain ones?

Reply:
If you think those would need added we can change it, but in my testing on google translate it seems like neither det nor den would be used for the other options, but I'm not sure if google translate can really be trusted :) .